indexing-service/src/main/java/io/druid/indexing/overlord/ThreadPoolTaskRunner.java

```diff
@@ -229,6 +239,7 @@ public TaskStatus call()
         status = TaskStatus.failure(task.getId());
       }
       catch (Throwable t) {
+        t.printStackTrace();
         log.error(t, "Uncaught Throwable while running task[%s]", task);
```

Comment:
You shouldn't need to call printStackTrace, the `log.error` should print the stack trace